redash/handlers/query_results.py

```diff
@@ -104,7 +104,7 @@ def post(self):
 
         :qparam string query: The query text to execute
         :qparam number query_id: The query object to update with the result (optional)
-        :qparam number max_age: If query results less than `max_age` seconds old are available, return them, otherwise execute the query; if omitted, always execute
+        :qparam number max_age: If query results less than `max_age` seconds old are available, return them, otherwise execute the query; if omitted or -1, returns any cached result, or executes if not available. Set to zero to always execute.
```

Comment:
Would you mind adding linebreaks to improve readability a bit? I think the best practice is the indent at the first character of the parameter description. Thanks!